src/main/index.ts

```diff
@@ -51,7 +50,7 @@ function createMainWindow() {
           }
 
           mainWindow
-              .loadURL(`http://localhost:${process.env.ELECTRON_WEBPACK_WDS_PORT}`)
+              .loadURL(`http://localhost:8080`) // todo pull from constants file
```

Comment:
checking intentional TODO left undone?

Reply:
Addressed in db8d371e09a2aecefe1ec07fefe59f8d1f668779 . Also highlighted the fact that one of my WebPack loaders was not in package.json 😬